source/elements/oneMath/source/domains/blas/blas-like-extensions.rst

```diff
@@ -11,8 +11,8 @@ BLAS-like Extensions
 .. container::
 
 
-   oneAPI Math Kernel Library DPC++ provides additional routines to
-   extend the functionality of the BLAS routines. These include routines
+   oneAPI Math Library DPC++ provides additional routines to extend
```

Comment:
Since we have `|onemath_full_name|`m  should we replace here:  `|onemath_full_name| (oneMath)`  ?  and maybe look for it elsewhere ?

Reply:
I updated them in [19c21e4](https://github.com/uxlfoundation/oneAPI-spec/pull/596/commits/19c21e49b8017fb635fd93c534752442aed33918) and, as far as I can tell, there aren't other occurrences of the full name